WordPress/Classes/Extensions/UIViewController+Notice.swift

```diff
@@ -33,3 +33,14 @@ extension UIViewController {
         ActionDispatcher.dispatch(NoticeAction.post(notice))
     }
 }
+
+@objc extension UIViewController {
+    @objc func displayNotice(title: String, message: String? = nil) {
```

Comment:
A possible improvement in the future is to include a `tag` so that `dismissNotice()` will only dismiss Notices _dispatched_ by the current ViewController.